src/algokit/core/typed_client_generation.py

```diff
@@ -98,6 +99,11 @@ def __init__(self) -> None:
         if not npx_path:
             raise click.ClickException("Typescript generator requires Node.js and npx to be installed.")
 
+        # Create the npm directory inside %APPDATA% if it doesn't exist, as npx on windows needs this.
+        if is_windows():
+            npm_dir = get_app_config_dir() / "npm"
+            npm_dir.mkdir() if not npm_dir.exists() else None
```

Comment:
This may fail due to permission errors. We need to catch and issue a console message indicating they are about to probably receive an error due to <link to github bug for npx> and they should create <folder> manually, since we couldn't for them.

Reply:
Yeah good idea.
We should be ok though, as it's the same root path as the algokit config directory.

Reply:
Fixed